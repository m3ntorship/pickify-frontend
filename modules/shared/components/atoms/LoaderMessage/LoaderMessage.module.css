.message {
  @apply text-lg sm:text-2xl md:text-3xl font-bold text-dark-grey whitespace-nowrap z-10 absolute top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2;
}
