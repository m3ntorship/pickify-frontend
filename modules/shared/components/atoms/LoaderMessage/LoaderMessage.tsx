import React from 'react';
import type { FC, ReactElement } from 'react';
import styles from './LoaderMessage.module.css';

const LoaderMessage: FC = ({ children }): ReactElement => {
  return <h1 className={styles.message}>{children}</h1>;
};

export default LoaderMessage;
