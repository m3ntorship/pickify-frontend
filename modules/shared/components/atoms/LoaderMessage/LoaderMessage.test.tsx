import React from 'react';
import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import LoaderMessage from './LoaderMessage';

describe('Loader Message', () => {
  it('Expect the loader message to toMatchSnapshot', () => {
    const tree = renderer
      .create(<LoaderMessage>Coming Soon</LoaderMessage>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Should return "Coming soon" as the the loader message ', () => {
    render(<LoaderMessage>Coming Soon</LoaderMessage>);
    const text = screen.getByText('Coming Soon');
    expect(text).toBeInTheDocument();
  });
});
