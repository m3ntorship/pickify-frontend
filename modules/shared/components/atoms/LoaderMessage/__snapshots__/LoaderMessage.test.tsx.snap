// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Message Expect the loader message to toMatchSnapshot 1`] = `
<h1
  className="message"
>
  Coming Soon
</h1>
`;
